desktop.ini

```diff
@@ -0,0 +1,4 @@
+[ViewState]
```

Comment:
Not required.

Reply:
@zeel991 This file should not be part of PR. Please remove this.

Reply:
I will recommend to do below:
- Copy the changes made in file pallets/network-score/src/tests.rs
- Remove all other changes by `git reset --hard HEAD~1`
- Apply the copied changes to same location.
- Add that file with `git add .`
- Amend the commit with `git commit --amend`
- Push to same branch, so we have a clean PR with only tests.rs changes.

Reply:
I deeply apologize for all this trouble , Thank you so much for guiding me !
I will follow these steps, and push it to the branch ,Thanks again .